src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  list-style-type: none;
+
+  &__star {
+    background: url(../images/star.svg) center no-repeat;
+    height: 16px;
+    width: 16px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  @for $value from 1 through 5 {
+    &--#{$value} :nth-child(-n + #{$value}) {
```

Comment:
There is an extra space between `&--#{$value}` and `:nth-child(-n + #{$value})`. According to the code style rules, there should be no space between the element and the pseudo-class.